lab_2_keywords_cooccurrence/main.py

```diff
@@ -30,13 +29,28 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    if not type_check(text, str):
+
+    if not isinstance(text, str) or len(text) == 0:
```

Comment:
You can simplify this statement, don't need to check the length, just use truthy or falsy check. For example:
`if not isinstance(text, str) or not text:`